subsys/mgmt/hawkbit/Kconfig

```
@@ -2,7 +2,7 @@
 # SPDX -License-Identifier: Apache-2.0
 
 menuconfig HAWKBIT
-	bool "Hawkbit Firmware Over-the-Air support"
+	bool "hawkBit Firmware Over-the-Air support"
```

albeit a bit tricky, let's try to comply with the Eclipse Foundation Trademark Usage Policy and change this one as follows:

```suggestion
	bool "Eclipse hawkBit Firmware Over-the-Air support"
```